src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -234,6 +234,8 @@ Start, Stop, Restart
 
 .. practical::
 
+   .. TODO - is this Met Office specific?
```

Comment:
???

Reply:
> In the bin directory is a script called get-observations. This script gets weather data from the MetOffice DataPoint service. 

Also 

```console 
$ rose tutorial runtime-tutorial

$ rose tutorial api-key
```

So not really a standalone Cylc practical??

and

Reply:
All of the tutorials used the `rose tutorial` command to safe people having to set up directories, copy files and all that stuff. For the "runtime" tutorial copying the files manually would be far too slow.

When we transferred the tutorials from Rose we transferred the tutorial files into this repo too, however, we do not presently have a Cylc alternative to the `rose tutorial` command.

* The files are now in this repo e.g: https://github.com/cylc/cylc-doc/blob/master/cylc/doc/etc/tutorial/api-keys
* They can be accessed via GH-Pages e.g: https://cylc.github.io/cylc-doc/nightly/html/_static/tutorial/api-keys
* Or if cylc-doc is installed they are included in the package e.g: `Path(cylc.doc.__file__.parent) / 'etc/api-keys'`
* The related issue is here: https://github.com/cylc/cylc-doc/issues/40

> This script gets weather data from the MetOffice DataPoint service

The DataPoint service was used for the Cylc forecasting tutorial, API keys for accessing the service are bundled with the suite so it is accessible to all. At the time DataPoint was the best offering that we could use without requiring users to setup accounts and so on. We will need to replace DataPoint soon as it is deprecated. We need point wind observations and a rainfall radar-like product. WOW may provide an answer but would require users to setup API keys etc.